Song/Koballoon.i7x

```diff
@@ -170,14 +170,26 @@ to say KoballoonHyperLoss: [requires Twisted Capacity]
 			say "     Fortunately they are perfectly happy to help you along, and continue to ravish you with all of their considerable energy and size. Due to your experience at being treated like a vessel for them to vent their lusts into, you're able to take it all and keep going happily, even if the sheer constant barrage of pleasure starts to fray the edges of your awareness. The sensations are enough for you to sink deeper into a haze of sex, each moment of intense feeling blurring into the next. During the extent of your ravaging the koballoons pump several loads of essence into you, perhaps even enough that they traded places with others that were still raring to go, but by that point your perception going no further than each of the cocks, leaving you unaware of the specifics.";
 			say "     Finally they reach the end of their own relief, pulling out completely and to much applause from the normal koballoons still watching the show. Your holes are no longer sealed with their shafts, but the loads they've left behind are doing more than a good enough job at keeping you plugged up and pumped up, at least for now. The group starts to head off, a few catcalls and comments about the next time all they leave with you. In the end, they wander off completely, leaving you little more than a cum-drenched balloon. As your mind returns to full awareness, you realize that if the koballons are creating more of these hyper versions of themselves, their need for service is only going to grow. If they take you again, they may find themselves using you so much that you never return to being anything other than their fuck blimp—and you're starting to think there might not be anything wrong with that at all.";
 			[increase KoballoonHyperTracker by 1;]
```

Comment:
This needs to be uncommented (aka remove the `[ ]` surrounding it, to make it work.